### PR TITLE
formplayer changes related to list query data support

### PR DESCRIPTION
**Pull request by snopoke**
Cross request from https://github.com/dimagi/commcare-core/pull/1078

There are no function changes here and no support for the list query data type either since that is blocked by the code to produce the 'selected-cases' instance.

### CODE REVIEW

File: src/main/java/org/commcare/formplayer/util/PrototypeUtils.java [hunk 1]
```diff
@@ -138,6 +140,8 @@ public static void setupThreadLocalPrototypes() {
                 BubbleSeries.class.getName(),
                 SendAction.class.getName(),
                 XPathDistinctValuesFunc.class.getName(),
+                ValueQueryData.class.getName(),
+                ListQueryData.class.getName(),
```
Comment (snopoke) [inline src/main/java/org/commcare/formplayer/util/PrototypeUtils.java:144]: Is this right? I wasn't sure but it seems to work.

Comment (orangejenny) [reply src/main/java/org/commcare/formplayer/util/PrototypeUtils.java:144]: I don't really know how this works. If this change is removed, does something break?

**Review by orangejenny — APPROVED** [review]
I'm not 100% clear on the PrototypeUtils change, but I also don't think it's risky.

File: src/main/java/org/commcare/formplayer/util/PrototypeUtils.java (hunk 1 above)
Comment (shubham1g5) [reply src/main/java/org/commcare/formplayer/util/PrototypeUtils.java:144]: yes, looks good. Serialization engine needs all implementations of `Externalizable` to be added here to be able to [instantiate classes](https://github.com/dimagi/commcare-core/blob/master/src/main/java/org/javarosa/core/util/externalizable/PrototypeFactory.java#L135) correctly while deserializing.

**Review by shubham1g5 — APPROVED** [review]

**Pull request merged**